source/fluid/backend/raylib5.d

```diff
@@ -398,14 +398,14 @@ class Raylib5Backend : FluidBackend {
 
     }
 
-    private fluid.backend.Texture fromRaylib(raylib.Texture texture, fluid.backend.Image.Format format) {
+    public fluid.backend.Texture toFluid(raylib.Texture rayTexture, fluid.backend.Image.Format format) @system {
```

Comment:
Don't make this `public`. 

Perhaps it might be a good idea to give this a different name from the global `toFluid` to avoid confusion. `moveToFluid` would be good.

Reply:
I'm against the renaming. I think it's better to have two functions that do the same thing have the same name. Confusion is unlikely, especially if a comment is added to warn of the other function with the same name.

But if you really want it changed, I think fromRaylib would be better for the class function.

Reply:
I ended up changing them back to `fromRaylib`.